mythril/mythx/__init__.py

```diff
@@ -30,6 +33,11 @@ def analyze(contracts: List[SolidityContract], analysis_mode: str = "quick") ->
         password=os.environ.get("MYTHX_PASSWORD", "trial"),
     )
 
+    if c.eth_address == "0x0000000000000000000000000000000000000000":
```

Comment:
using a variable for that 0x000.. in here and above might be cleaner.

Reply:
:+1: